src/common/services/request/query/fetch-plays-filter.js

```diff
@@ -14,7 +14,7 @@ function createObjectPayload(items, key, ifTags) {
   if (items.length > 1) {
     obj.clause = {
       operator: 'or',
-      conditions: !env && !preview ? [defaultClause] : []
+      conditions: []
```

Comment:
@Angryman18 Please check why the condition was like that? Based on your feedback, we will merge this change.

Reply:
that env var not defined in local .env and moreover that variable is not being used in anywhere in our app since i have just searched it widely and found no match. and anther thing is i think we clean this section a little bit more since we donot need these preview mode and also that default clause for dev_mode which has no more usecase i guess. so we can completely remove the defaultClause and keep conditions as blank array.

Reply:
Thanks for confirming @Angryman18 . I agree with you. In that case, @supminn 's fix looks good. Please approve the PR if it looks good to you too.